src/backend/core/authentication/exceptions.py

```diff
@@ -0,0 +1,5 @@
+"""Exceptions for the authentication module."""
+
+
+class EmailNotWhitelisted(Exception):
```

Comment:
```suggestion
class EmailNotAuthorized(Exception):
```

Reply:
The specific name is on purpose to allow to make a specific redirection in the backend ( `?auth_error=alpha` to keep track of the real reason )

Reply:
You can rename the exception while redirecting with this specific error. I don't see why you can't rename it.

Reply:
or you can use `EmailNotAllowed`.

See https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white